nixos/modules/services/security/vault.nix

```
@@ -186,6 +207,9 @@ in
         Group = "vault";
         ExecStart = "${cfg.package}/bin/vault server ${configOptions}";
         ExecReload = "${pkgs.coreutils}/bin/kill -SIGHUP $MAINPID";
+        StateDirectory = "vault";
+        # In `dev` mode vault will put it's token here
```

```suggestion
        # In `dev` mode vault will put its token here
```

English has no rules, only exceptions.

---

Should `StateDirectory` be set when `dev = false`?